doc/presentations.rst

```diff
@@ -24,21 +52,44 @@ specific subject areas:
 Videos
 ======
 
-- An introduction to scikit-learn `Part
-  I <https://conference.scipy.org/scipy2013/tutorial_detail.php?id=107>`_ and
-  `Part II <https://conference.scipy.org/scipy2013/tutorial_detail.php?id=111>`_ at Scipy 2013
-  by `Gael Varoquaux`_, `Jake Vanderplas`_  and `Olivier Grisel`_. Notebooks on
+- The `:probabl YouTube channel <https://www.youtube.com/@probabl_ai>`_
+  hosts the Whiteboard Series, which provides supplementary materials for
```

Comment:
```suggestion
  hosts the `Whiteboard Series <https://www.youtube.com/playlist?list=PLSIzlWDI17bS025ph6R0W_3RKM0qJ3qoO>`_,
  which provides supplementary materials for
```

again just because the URL is not comprehensible, but also open to showing full URL